### ISSUE TITLE
Error on expo start

### ISSUE THREAD

**Issue by jsiddharth** [GeekyAnts/NativeBase-KitchenSink#196]
INFO
21:07
Starting Metro Bundler
ERROR
21:08
Unable to resolve "AccessibilityInfo" from "node_modules/react-native/Libraries/react-native/react-native-implementation.js"
ERROR
21:08
Building JavaScript bundle: error
